packages/ethers-ccip-read-provider/src/index.ts

```diff
@@ -106,7 +106,7 @@ async function sendRPC(fetcher: Fetch, urls: string[], to: BytesLike, callData:
     const url = template.replace(/\{([^}]*)\}/g, (_match, p1: keyof typeof args) => args[p1]);
     const data = await fetcher(url, template.includes('{data}') ? undefined : JSON.stringify(args), processFunc);
     if (data.status >= 400 && data.status <= 499) {
-      return logger.throwError('bad response', Logger.errors.SERVER_ERROR, {
+      logger.warn('Bad response', Logger.errors.SERVER_ERROR, {
```

Comment:
This will both log this error and the one below, and fall through to the next iteration of the loop. I don't think that's what you intended?

Reply:
right, it's not what I was intended. updated now.